### PR TITLE
MNT: Re-rendered with conda-smithy 2.0.1

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
Reference: https://github.com/conda-forge/conda-smithy/issues/450#issuecomment-276864564

File: .travis.yml [hunk 1]
```diff
@@ -4,33 +4,47 @@
...
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
+    - CONDA_PY=36
```
Comment (ocefpaf) [inline .travis.yml:14]: :+1:

Comment (ericmjl) [reply .travis.yml:14]: Wondering if there's already an automated way to get all of the feedstocks re-rendered with the latest conda-smithy?

**Comment by ericmjl** [comment]
haha, as usual, Travis is the slowest! :smile:

**Comment by ocefpaf** [comment]
Actually they had a major outage in the past days. We can merge this without waiting for it. Bare in mind that the OS X packages will be pending though.

**Comment by ericmjl** [comment]
@ocefpaf didn't realize that we could merge without waiting! For my education purposes, can I check, what goes on after merging, that we're allowed to merge without waiting for Travis?

**Comment by ocefpaf** [comment]
> @ocefpaf didn't realize that we could merge without waiting!

Please do not do that. It is a bad practice and the only reason I did it here is because Travis-CI is awfully slow these past days, and I am monitoring the `OS X` build on master.

> For my education purposes, can I check, what goes on after merging, that we're allowed to merge without waiting for Travis?

The package will be built again and will be uploaded. You can follow the CI that I "by-passed" here: https://travis-ci.org/conda-forge/pygithub-feedstock/builds/197721710

**Comment by ericmjl** [comment]
@ocefpaf got it! Thanks a ton!

**Pull request merged**